server/views/pages/approve/cases.njk

```diff
@@ -184,7 +184,7 @@
             {
                 text: "Release date",
                 attributes: {
-                    "aria-sort": "ascending" if approvalNeededView else "none"
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏The release date is now sorted from the backend so this is not needed.